content/events/2020/10/2020-10-22-women-in-federal-it-cyber-conference-panel-discussion-3-veterans-day.md

```diff
@@ -0,0 +1,44 @@
+---
+title: "Women In Federal IT & Cyber Conference Panel Discussion #3: Veterans Day"
+kicker: CIO Council
+summary: Panelists will discuss the challenges of transitioning and the benefits
+  of transitioning from military to civilian service in the IT field.
+host: CIO Council
+event_organizer: Digital.gov
+registration_url: "https://www.eventbrite.com/e/women-in-federal-it-cyber-confe\
+  rence-panel-discussion-3-veterans-day-tickets-126324226161 "
+captions: "https://www.captionedtext.com/client/event.aspx?EventID=4613207&Cust\
+  omerID=321 "
+date: 2020-11-10 14:00:00 -0500
+end_date: 2020-11-10 15:00:00 -0500
+# See all topics at https://digital.gov/topics
+topics:
+  - professional-development
+# See all authors at https://digital.gov/authors
+authors:
+  - renata-spinks
+  - donna-roy
+slug: women-in-federal-it-cyber-conference-panel-discussion-3-veterans-day
+# zoom, youtube_live, adobe_connect, google
+event_platform: zoom
+youtube_id: "null"
+---
+A networking event complementing the video to be released on Monday, November 10, Service to Service, on the perspectives from women who have transitioned from serving in the armed forces to serving our country as civilian government employees. In this networking event, participants will have the opportunity to ask questions to senior leaders who have transitioned from military to civilian service in the IT and cyber field. Come prepare to ask on transitioning, leadership and career development.
```

Comment:
typo fix, should be "Come prepared"

Reply:
also: should it be "Come prepared to ask **questions** on transitioning..." 
this reads a bit weird to me

Reply:
I removed the sentence completely. It seemed redundant to me.